technologies/job/spark/spark-3.0/entrypoint.sh

```diff
@@ -30,18 +30,25 @@ set -e
 # If there is no passwd entry for the container UID, attempt to create one
 if [ -z "$uidentry" ] ; then
     if [ -w /etc/passwd ] ; then
-        echo "$myuid:x:$myuid:$mygid:anonymous uid:$SPARK_HOME:/bin/false" >> /etc/passwd
+        echo "$myuid:x:$myuid:$mygid:${SPARK_USER_NAME:-anonymous uid}:$SPARK_HOME:/bin/false" >> /etc/passwd
```

Comment:
Is it necessary ? (not added in the PR description)

Reply:
It comes from spark code : https://github.com/apache/spark/blob/a7d0d353cd745391cf285039a4630b37754a56a0/resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh